File: igor/Problema0057/problema0057.py
quantidadeFitas = int(input("Quantidade de fitas: "))
valorAluguel = float(input("Valor aluguel: "))

faturamentoAnual = ((int(quantidadeFitas/3)) * valorAluguel)*12
print("Faturamento Anual: ",faturamentoAnual)

totalMultas = (quantidadeFitas/10) * (valorAluguel*(110/100))
print("Valor ganho por multa ao mes: ",totalMultas)

reposicao = quantidadeFitas * (10/100)
fitasEstragadas = quantidadeFitas * (2/100)
quantidadeFinal = quantidadeFitas - fitasEstragadas + reposicao
print("Quantidade de fitas final: ",quantidadeFinal)

File: igor/Problema0057/problema0057.java
import java.util.Scanner;

public class problema0057{

    static Scanner input = new Scanner(System.in);
    public static void main(String[] args) {
        
        System.out.println("Quantidade de Fitas: ");
        int quantidadeFitas = input.nextInt();
        System.out.println("Valor aluguel: ");
        float valorAluguel = input.nextFloat();

        float faturamentoAnual = ((quantidadeFitas/3)*valorAluguel)*12;
        System.out.println("Faturamento Anual: "+faturamentoAnual);

        float totalMultas = (quantidadeFitas/10) * ((valorAluguel * 110)/100);
        System.out.println("Valor ganho por multa ao mes: "+totalMultas);

        
        int reposicao = (quantidadeFitas * 10)/100;
        int fitasEstragadas = (quantidadeFitas  * 2)/100;
        int quantidadeFinal = quantidadeFitas - fitasEstragadas + reposicao;
        System.out.println("Quantidade de fitas final: "+quantidadeFinal);


    }


}